data/templates/post-windows-install.ps1

```diff
@@ -0,0 +1,30 @@
+<% if( undefined !== networkDevices ) { %>
+            <% networkDevices.forEach(function(n) { %>
+                  <% if( undefined != n.ipv4.vlanIds ) { %>
+                        Get-NetAdapterAdvancedProperty <%= n.device%> -RegistryKeyword "VlanID"
+                        If($? -eq "0") #Make sure that VLAN is supported  on a the specified Ethernet port
+
+                            {
+                                Write-Host "Vlan is supported on <%= n.device%> Ethernet port"
+                                IF(<%=n.ipv4.vlanIds[0]%> -ge 0 -Or <%=n.ipv4.vlanIds[0]%> -le 4095)
+                                    {
+                                        Set-NetAdapterAdvancedProperty <%= n.device%> -RegistryKeyword "VlanID" -DisplayValue <%=n.ipv4.vlanIds[0]%>
+                                    }
+                                Else
+                                    {
+                                        Write-Host "Vlan value should with the 0-4095 range"
+                                    }
+                            }
+                        Else
+                            {
+                                Write-Host "Vlan is NOT supported on <%= n.device%> Ethernet port"
+                            }
+
+                   <% }%>
+
+           <%});%>
+<% } %>
+
+
+#crul the renasar-ansible.pub to indicate that windows install workflow has completed
+curl http://<%=server%>:<%=port%>/api/2.0/templates/renasar-ansible.pub -Outfile renasar-ansible.pub

```

Comment:
Hi Felix, I am  going to keep, as it is for now.
I have a story to convert it later.
https://www.pivotaltracker.com/story/show/127112215